Source/itemdat.cpp

```diff
@@ -226,94 +226,94 @@ const PLStruct ItemPrefixes[] = {
 	// clang-format off
 	// PLName,              { type,                 param1, param2 }, PLMinLvl, PLIType,                                                         PLGOE,    PLDouble, PLOk,  minVal, maxVal, multVal

```

Comment:
Probably the header needs some adjustment now?

Reply:
missed that - fixed